src/Microsoft.AspNetCore.Mvc.Core/Internal/MvcCoreLoggerExtensions.cs

```diff
@@ -289,6 +293,10 @@ static MvcCoreLoggerExtensions()
                 1,
                 "The request has model state errors, returning an error response.");
 
+            _inferringParameterSource = LoggerMessage.Define<MethodInfo, string, string>(
+                LogLevel.Debug,
+                1,
+                "Inferring binding source for '{ActionName}.{ParameterName} as {BindingSource}.");
```

Comment:
`Inferred`

Reply:
`'{ActionName}.{ParameterName` - this is missing a closing quote. Also this isn't a good syntax - no one expects `Foo.Bar` to mean parameter